src/edu/csus/ecs/pc2/core/report/FileComparisonUtilities.java

```diff
@@ -0,0 +1,500 @@
+                fileComparison.addfieldCompareRecord(fieldCompareRecord);
+
+                fieldName = "team_id";
+                valueOne = Long.toString(firstRow.getTeam_id());
+                fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, valueOne);
+                fileComparison.addfieldCompareRecord(fieldCompareRecord);
+
+                // TODO 760 compare team names
+                //                fieldName = "team name";
+                //                valueOne = firstRow.getTeamName();
+                //                fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, valueOne);
+                //                fileComparison.addfieldCompareRecord(fieldCompareRecord);
+            }
+            
+            for (int rowNum = minLines; rowNum < firstTeamScoreRows.size(); rowNum++) {
+
+                numberRows ++;
+                
+                TeamScoreRow teamScoreRow = firstTeamScoreRows.get(rowNum);
+                
+                String valueOne = null;
+
+                String fieldName = "rank";
+                String valueTwo = Integer.toString(teamScoreRow.getRank());
+                FieldCompareRecord fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, valueOne);
+                fileComparison.addfieldCompareRecord(fieldCompareRecord);
+
+                // TODO 760 Do we need to compare problems??  firstRowfirstRow.getProblems() ?
+                fieldName = "num solved";
+                valueTwo = Long.toString(teamScoreRow.getScore().getNum_solved());
+                fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, valueOne);
+                fileComparison.addfieldCompareRecord(fieldCompareRecord);
+
+                fieldName = "total_time";
+                valueTwo = Long.toString(teamScoreRow.getScore().getTotal_time());
+                fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, valueOne);
+                fileComparison.addfieldCompareRecord(fieldCompareRecord);
+
+                fieldName = "team_id";
+                valueTwo = Long.toString(teamScoreRow.getTeam_id());
+                fieldCompareRecord = new FieldCompareRecord(fieldName, valueOne, valueTwo, null, valueOne);
+                fileComparison.addfieldCompareRecord(fieldCompareRecord);
+
+            }
+        } catch (Exception e) {
+            StaticLog.getLog().log(Level.WARNING, "Problem comparing scoreboard data/info", e);
+            e.printStackTrace(); // TODO 760 how to report problem to user ?
+        }
+
+        fileComparison.setNumberRows(numberRows);
+
+        return fileComparison;
+
+    }
+//   
+
+    public static FileComparison createAwardJSONFileComparison(String tsvFileName, String sourceDir, String targetDir, IFileComparisonKey fileComparisonKey) {
```

Comment:
Same complaint as above: The parameter name `tsvFileName` is _very_ misleading, since this method is _never_ called (and should never BE called) with a "TSV" file -- it is, and should only be -- called with a JSON file.

Reply:
parameter name updated.